dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializerTest.kt

```diff
@@ -80,13 +79,28 @@ internal class RumEventSerializerTest {
             }
             .hasField("_dd") {
                 hasField("format_version", 2L)
+                hasNullableField("span_id", event.dd.spanId)
+                hasNullableField("trace_id", event.dd.traceId)
             }
-        event.usr?.let {
+
+        event.usr?.let { usr ->
             assertThat(jsonObject).hasField("usr") {
-                hasNullableField("id", event.usr?.id)
-                hasNullableField("name", event.usr?.name)
-                hasNullableField("email", event.usr?.email)
-                containsAttributes(it.additionalProperties)
+                hasNullableField("id", usr.id)
+                hasNullableField("name", usr.name)
+                hasNullableField("email", usr.email)
+                containsAttributes(usr.additionalProperties)
+            }
+        }
+        event.connectivity?.let { connectivity ->
+            assertThat(jsonObject).hasField("connectivity") {
+                hasNullableField("status", connectivity.status.name.lowercase())
```

Comment:
`lowercase` without a `Locale` doesn't raise a warning?

Reply:
no it doesn't but I'll add that indeed